addons/api/fnc_getVOIPUID.sqf

```diff
@@ -0,0 +1,18 @@
+#include "script_component.hpp"
+/*
+ * Author: Killerswin2
+ * Get the VOIP UID.
```

Comment:
UID of what? Specify.

If this is TS3 client UID, why are we duplicating it? We already have client ID in `getClientID` RPC, available via `EGVAR(sys_core,ts3id)`. I see no reason to expose that though.

Server UID and Channel UID would be far more useful.

Reply:
Yes this the TS3 client UID, are you asking for the remove of the rpc to UID and adding both server uid and channel uid?

Reply:
This returns the SERVER UID, not the TS3 client UID.

Reply:
I'm also going to keep the VOIP because of future Mumble support.